src/Configuration/ValueObject/SnowflakeDatabaseConfig.php

```diff
@@ -32,7 +32,7 @@ public static function fromArray(array $data): DatabaseConfig
             $data['warehouse'] ?? null,
             $data['roleName'] ?? null,
             $sslEnabled ? SSLConnectionConfig::fromArray($data['ssl']) : null,
-            $data['#keyPair'] ?? null,
+            $data['#keyPair'] ?? $data['#privateKey'] ?? null,
```

Comment:
```suggestion
            $data['#privateKey'] ?? $data['#keyPair'] ?? null,
```
řekl bych že tady by to mělo byt takto ať má přednost 'privateKey'. Mám config tam mám keyPair UI sa změní na privateKey nastavím privateKey, ale komponenta bude pořád brat keyPair field, který už v UI nemám. (nebo třeba nevím jak to funguje a kdy sa toto volá :D )

Reply:
jo mas pravdu takto som sa nad tym nezamyslel tru .. Ď